zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/cacheAPI/SendNotificationCacheApi.java

```diff
@@ -0,0 +1,43 @@
+package com.github.onsdigital.zebedee.model.publishing.cacheAPI;
+
+import com.github.davidcarboni.httpino.*;
+import com.github.onsdigital.zebedee.configuration.Configuration;
+import com.github.onsdigital.zebedee.json.EventType;
+import com.github.onsdigital.zebedee.model.publishing.PublishNotification;
+import com.github.onsdigital.zebedee.model.publishing.WebsiteResponse;
+import com.github.onsdigital.zebedee.util.EncryptionUtils;
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
+
+import java.io.IOException;
+import java.util.List;
+
+import static com.github.onsdigital.logging.v2.event.SimpleEvent.info;
+
+public class SendNotificationCacheApi {
+    private static final String LEGACY_CACHE_API_PATH = "/v1/cache-times/";
+
+    public SendNotificationCacheApi() {
+    }
+
+    public void sendNotification(Http http, Host host, EventType eventType, List<NotificationPayloadCacheApi> payloads, String collectionId) throws IOException {
```

Comment:
A couple of small changes:
1. I think you can move the `host` to the constructor, as that one is going to always be the same, it won't change with different requests.
2. Isn't the `eventType` in our case always "publish"? Why would we want to pass it as a parameter?